wye/workshops/urls.py

```diff
@@ -10,4 +11,6 @@
         WorkshopUpdate.as_view(), name="workshop_update"),
     url(r'^(?P<action>[active,deactive]+)/(?P<pk>\d+)/$',
         WorkshopToggleActive.as_view(), name="workshop_toggle"),
+    url(r'^presenter/(?P<action>[opt\-in,opt\-out]+)/(?P<pk>\d+)/$',
```

Comment:
Is it ok, if I update the urls pattern for active/deactive workshop as below, So that urls will be in sync.

From: workshop/active/1
To: workshop/1/active